### PR TITLE
Portfolio: Setup and Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by julie-ify**
## Portfolio mobile version

In the first milestone in the process of creating my portfolio website, I:
- opened a new repository and set it up for deployment using best practices.
- Added linters config files for fixing validation errors.
- Built the first 3 sections of the mobile version of the website using template from Figma.
- Used HTML5 semantic elements to improve the structure and accessibility of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Portfolio Mobile Version</title>
+  </head>
+  <body>
+    <main>
+      <header class="header-content">
+        <h2 class="brand">Welcome</h2>
```
Comment (rloterh) [inline index.html:13]: - [ ] Your logo is not displaying as a link. Please ensure to add a link tag to the logo by using `a` tag.
-

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Juliana, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Highly organized/clean code
- Great UI design
- Good ReadMe
- No linter errors


## Required Changes ♻️
- [ ] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

File: index.html (hunk 1 above)
Comment (julie-ify) [reply index.html:13]: Ok

**Review by mikenath223 — APPROVED** [review]
Hello there,

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**